skops/cli/tests/test_convert.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import pathlib
+import pickle as pkl
```

Comment:
```suggestion
import pickle
```

for consistency with the rest of the code base.